rpc.go

```diff
@@ -115,7 +124,7 @@ func (r rpc) Channel(service string) transport.Channel {
 func (r rpc) Start() error {
 	callServe := func(i transport.Inbound) func() error {
 		return func() error {
-			return i.Start(r)
+			return i.Start(r, r)
```

Comment:
Before anyone asks, this is passing the RPC object twice, but as the Handler and Dependencies interface separately. The Dependencies will also be threaded through outbounds. We could alternately mix the Dependencies interface into the Handler interface.